tools/export-dot/export-dot.cpp

```diff
@@ -277,7 +277,8 @@ static StringRef getNodeColor(Operation *op) {
   return llvm::TypeSwitch<Operation *, StringRef>(op)
       .Case<handshake::ForkOp, handshake::LazyForkOp, handshake::JoinOp>(
           [&](auto) { return "lavender"; })
-      .Case<handshake::BufferOp>([&](auto) { return "palegreen"; })
+      .Case<handshake::BlockerOp>([&](auto) { return "cyan"; })
+      .Case<handshake::BufferOp>([&](auto) { return "green"; })
```

Comment:
Is the reason to change the color of buffer to make two colors more clearly separated? Can you only adjust the color of the blocker, or was it difficult?